### PR TITLE
ZBUG-3911 : Create rpm/debian packages for zimbra-zimlet-sticky-notes zimlet

### PR DESCRIPTION
**Pull request by umagmrit**

### CODE REVIEW

File: scripts/postinst.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+echo "*** Configuring Zimbra-Zimlet-Sticky-Notes ***"
```
Comment (jiteshsojitra) [inline scripts/postinst.sh:2]: one blank line

File: scripts/postinst.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+su - zimbra -c "zmmailboxdctl status"
+if [ $? -ne 0 ]; then
```
Comment (jiteshsojitra) [inline scripts/postinst.sh:4]: one blank line

File: scripts/postinst.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+   echo "*** Follow the steps below as zimbra user ignore if installing through install.sh .. ***"
+   echo "*** Install the Zimbra-Zimlet-Sticky-Notes zimlet. ***"
+   echo "*** zmzimletctl -l deploy /opt/zimbra/zimlets-network/zimbra-zimlet-sticky-notes.zip ***"
+   echo "*** zmprov fc zimlet ***"
+else
+   echo "*** Deploying Zimbra-Zimlet-Sticky-Notes zimlet ***"
+   su - zimbra -c  "zmzimletctl -l deploy /opt/zimbra/zimlets-network/zimbra-zimlet-sticky-notes.zip"
+   su - zimbra -c  "zmprov fc zimlet"
+fi
+echo "*** Zimbra-Zimlet-Sticky-Notes Installation Completed. ***"
```
Comment (jiteshsojitra) [inline scripts/postinst.sh:15]: one blank line

File: scripts/postinst.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+echo "*** Restart the mailbox service as zimbra user. Run ***"
+echo "*** su - zimbra ***"
+echo "*** zmmailboxdctl restart ***"
```
Comment (jiteshsojitra) [inline scripts/postinst.sh]:
```
#!/bin/bash

packageName="Zimbra-Zimlet-Sticky-Notes"
packageZipName="zimbra-zimlet-sticky-notes.zip"

echo "*** Configuring ${packageName} ***"

su - zimbra -c "zmmailboxdctl status"
if [ $? -ne 0 ]; then
   echo "*** Mailbox is not running... ***"
   echo "*** Follow the steps below as zimbra user ignore if installing through install.sh .. ***"
   echo "*** Install the ${packageName} zimlet. ***"
   echo "*** zmzimletctl -l deploy /opt/zimbra/zimlets-network/${packageZipName} ***"
   echo "*** zmprov fc zimlet ***"
else
   echo "*** Deploying ${packageName} zimlet ***"
   su - zimbra -c  "zmzimletctl -l deploy /opt/zimbra/zimlets-network/${packageZipName}"
   su - zimbra -c  "zmprov fc zimlet"
fi

echo "*** ${packageName} Installation Completed. ***"
echo "*** Restart the mailbox service as zimbra user. Run ***"
echo "*** su - zimbra ***"
echo "*** zmmailboxdctl restart ***"
```

Comment (umagmrit) [reply scripts/postinst.sh]: updated it

**Review by jiteshsojitra — APPROVED** [review]

File: .circleci/config.yml [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+############################################################################
+
+references:
+   checkout_job_steps: &checkout_job_steps
+      steps:
+         - checkout
+         - run: 
+            name: Checking out dependencies
+            command: |
+               git clone https://github.com/Zimbra/zm-pkg-tool.git ~/zm-pkg-tool
```
Comment (silentsakky) [inline .circleci/config.yml]: `--depth 1`

Comment (umagmrit) [reply .circleci/config.yml]: done

**Review by silentsakky — APPROVED** [review]

**Pull request merged**